source/wp-content/themes/wporg-showcase-2022/src/site-screenshot/index.php

```diff
@@ -53,6 +53,11 @@ function render( $attributes, $content, $block ) {
 
 	$img_content = "<img src='{$screenshot}' srcset='$srcset 2x' alt='" . the_title_attribute( array( 'echo' => false ) ) . "' />";
 
+	if ( isset( $attributes['isLink'] ) && true == $attributes['isLink'] ) {
+		$img_content .= '<span class="screen-reader-text">' . the_title_attribute( array( 'echo' => false ) ) . '</span>';
```

Comment:
Since the image already has this text as the alt attribute, this creates a duplicate for the link's text, ex "Childrens Radio Foundation Childrens Radio Foundation"

Reply:
I'll remove it.